data/test/index.js

```diff
@@ -32,6 +33,11 @@ jest.mock( '@wordpress/utils', () => ( {
 	deprecated: jest.fn(),
 } ) );
 
+// Mock data store to prevent self-initialization, as it needs to be reset
+// between tests of `registerResolvers` by replacement (new `registerStore`).
+jest.mock( '../store', () => () => {} );
+const registerDataStore = require.requireActual( '../store' ).default;
```

Comment:
We probably should move out all methods from the `index.js` to avoid this kind of mocking.